File: ProyectoFinal/ProyectoFinal/Parametros/Salida/ListarUsuariosOut.cs
﻿using ProyectoFinal._2_Dominio;
using System;
using System.Collections.Generic;
using System.Linq;
using System.Web;

namespace ProyectoFinal.Parametros.Salida
{
    public class ListarUsuariosOut : ParametroBaseOut
    {
        public List<Usuario> Usuarios { get; set; }
    }
}

File: Interfaz/ProyectoFinal/src/app/Parametros/Salida/ListarUsuariosOut.ts
import { Usuario } from "src/app/Components/Usuario/class/usuario";
import { ParametroBaseOut } from "./ParametroBaseOut";

export class ListarUsuariosOut extends ParametroBaseOut{

    private usuarios: Usuario[] = [];

    constructor()
    {
        super();
    }

    public get Usuarios(): Usuario[] {
        return this.usuarios;
    }
    public set Usuarios(value: Usuario[]) {
        this.usuarios = value;
    }
}